src/main/java/frc/robot/RobotContainer.java

```diff
@@ -98,7 +100,10 @@ public RobotContainer() {
                                 m_drivetrain));
 
                 // Configure the trigger bindings
-                configureBindings();
+                configureBindings(); 
+
+                SmartDashboard.putData("Autonomous Balance PID",new AutoBalancePID(m_drivetrain));
```

Comment:
@mikerada6 here's how to add a command directly to the dashboard. Doing this causes a start button to pop up for the command, and while it's running there's a cancel button that you can push to stop it. It's really nice.